internal/scalibrextract/language/java/pomxmlenhanceable/pomxmlenhanceable.go

```diff
@@ -17,11 +17,15 @@ const (
 
 // Extractor extracts Maven packages from pom.xml files.
 type Extractor struct {
-	actual filesystem.Extractor
+	base     filesystem.Extractor
```

Comment:
nit: I'd probably call these `online` and `offline` since that's the heart of them